src/routes/[shortUrl]/stats/+page.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
```

Comment:
there should be invalidation on reload, I think its not nice to cache stats page (you make few clicks, go to stats page and its empty)

Reply:
I do invalidateAll in onMount on each dynamic data page - /shortUrl/stats and /list but nothing changed. I also check without cache in the browser and from  incognito tabs.  I think the problem is in Cloudflare KV,  the latency  from write to read is too big, and i read few post and KV docs where people said that it can be happed, maybe because of hot and cold reads.

So i think maybe Cloudflare KV is not perfect match for this kind of task or maybe it need to be used as a cache, not storage

https://developers.cloudflare.com/kv/reference/how-kv-works/
https://community.cloudflare.com/t/delay-in-workers-kv/149463